### DISCUSSION TITLE
3D rotation of a scatter marker

### DISCUSSION THREAD

**Discussion by dalarev** [JuliaPlots/Makie.jl#4323] [Q&A]
Hi, is it possible to rotate a scatter plot marker in 3 dimensions?

The [documentation](https://docs.makie.org/v0.21/reference/plots/scatter#Bezier-path-markers) shows only 2D examples, and I wasn't able to figure this out from the source code.

I am using the [uparrow](https://docs.makie.org/v0.21/reference/plots/scatter#Default-markers) scatter markers on an Axis3 surface, trying to rotate each arrow/marker in a direction normal to the surface at every point. I tried using the [arrows](https://docs.makie.org/v0.21/reference/plots/arrows) function for this but was having issues.

Appreciate any guidance you're able to offer. Thanks!

**Comment by SimonDanisch** [comment]
Yes, you can: 
```julia
scatter(rand(Point3f, 100), rotation=rand(Vec3f, 100))
```
